<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>380</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>556</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>556</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DyeTheWorld</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="originalButton">
        <property name="text">
         <string>Original</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="coloringPageButton">
        <property name="text">
         <string>Coloring Page</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="energyButton">
        <property name="text">
         <string>Energy</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QSlider" name="zoomSlider">
    <property name="geometry">
     <rect>
      <x>533</x>
      <y>0</y>
      <width>16</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>50</number>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>101</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="printButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Print</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Zoom:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="zoomSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QSlider" name="detailsSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ratio:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="detailsSpinBox">
         <property name="suffix">
          <string notr="true" comment="d" extracomment="d"/>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>409</width>
       <height>309</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>scrollArea</zorder>
   <zorder>zoomSlider</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>556</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>originalButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>displayModeToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>energyButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>displayModeToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>coloringPageButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>displayModeToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>detailsSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>detailsSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>detailsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>detailsSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zoomSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zoomSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>detailsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onDetailRatioChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onZoomChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displayModeToggled(bool)</slot>
  <slot>onDetailRatioChanged(int)</slot>
  <slot>onZoomChanged(int)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
